io-pay-portal-fe/src/helper.ts

```diff
@@ -40,6 +47,80 @@ export const getPaymentInfoTask = (
       )
   );
 
+export const activePaymentTask = (
+  organizationId: EnteBeneficiario,
+  amountSinglePayment: ImportoEuroCents,
+  paymentNoticeCode: CodiceContestoPagamento
+): TaskEither<string, PaymentActivationsPostResponse> =>
+  tryCatch(
+    () =>
+      apiClient.activatePayment({
+        body: {
+          rptId: `${organizationId}${paymentNoticeCode}`,
+          importoSingoloVersamento: amountSinglePayment,
+          codiceContestoPagamento: paymentNoticeCode,
+        },
+      }),
+    () => "Errore attivazione pagamento"
+  ).foldTaskEither(
+    (err) => fromLeft(err),
+    (errorOrResponse) =>
+      errorOrResponse.fold(
+        () => fromLeft("Errore attivazione pagamento"),
+        (responseType) =>
+          responseType.status !== 200
+            ? fromLeft(`Errore attivazione pagamento : ${responseType.status}`)
+            : taskEither.of(responseType.value)
+      )
+  );
+
+export const getActivationStatusTask = (
+  paymentNoticeCode: CodiceContestoPagamento
+): TaskEither<string, PaymentActivationsGetResponse> =>
+  tryCatch(
+    () =>
+      apiClient.getActivationStatus({
+        codiceContestoPagamento: paymentNoticeCode,
+      }),
+    () => "Errore stato pagamento"
+  ).foldTaskEither(
+    (err) => fromLeft(err),
+    (errorOrResponse) =>
+      errorOrResponse.fold(
+        () => fromLeft("Errore stato pagamento"),
+        (responseType) =>
+          responseType.status !== 200
+            ? fromLeft(`Errore stato pagamento : ${responseType.status}`)
+            : taskEither.of(responseType.value)
+      )
+  );
+
+export const pollingActivationStatus = async (
+  paymentNoticeCode: CodiceContestoPagamento,
+  attempts: number
+): Promise<void> => {
+  await getActivationStatusTask(paymentNoticeCode)
+    .fold(
+      () => {
+        // eslint-disable-next-line @typescript-eslint/no-unused-expressions
+        attempts > 0
+          ? setTimeout(
+              pollingActivationStatus,
+              getConfig("IO_PAY_PORTAL_PAY_WL_POLLING_INTERVAL") as Millisecond,
+              paymentNoticeCode,
+              --attempts // eslint-disable-line no-param-reassign
+            )
+          : showActivationError("Errore Attivazione Pagamento");
+      },
+      (acivationResponse) =>
```

Comment:
I'd add a comment says that here is the redirect to `io-pay` aka `wisp-light`
I think it is useful.